frontend/src/components/common/form/Check.scss

```diff
@@ -4,6 +4,6 @@
   align-items: center;
   padding: 0 5px;
   .form-check {
-    padding-right: 5px;
+    margin: 0px 5px;
```

Comment:
padding doesn't display properly in all cases, additionally need both left and right margin since the label can occur before or after the checkbox.